jnigen/test/simple_package_test/runtime_test_registrant.dart

```diff
@@ -595,15 +595,24 @@ void registerTests(String groupName, TestRunnerCallback test) {
         final manyPrimitives = await manyPrimitivesResult.future;
         expect(manyPrimitives, -1 + 3 + 3.14.toInt() + 1);
 
+        // Currently we have one implementation of the interface.
+        expect(MyInterface.$impls, hasLength(1));
         myInterface.delete();
+        // Running System.gc() and waiting for a second.
```

Comment:
This could be flaky. Maybe it would be less flaky and more speedy if we try it in a loop with 100 ms, 200 ms, ... 5 seconds?